### PR TITLE
Feature/fill in logging gaps

### PR DESCRIPTION
**Pull request by gemmai95**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -235,6 +235,7 @@ def do_get_response(mongo_id):
 
 @app.route('/queue', methods=['POST'])
 def do_queue():
+    logger.info("Begin Queueing")
```
Comment (necrophonic) [inline server.py:238]: Needs to be logging the `tx_id` - don't think it's bound to the logger instance you're using here

File: server.py [hunk 2]
```diff
@@ -253,6 +254,7 @@ def do_queue():
     if queued is False:
         return server_error("Unable to queue notification")
 
+    logger.info("Successfully Queued")
```
Comment (necrophonic) [inline server.py:257]: `tx_id` as above

File: server.py (hunk 1 above)
Comment (necrophonic) [reply server.py:238]: Also, what is that's being queued to where?

**Comment by insacuri** [comment]
This was taken off master (which is using mongo), another branch has been taken off develop (thats using postgres)

**Pull request closed without merging**